### PR TITLE
rgw: multisite enabled over multiple clusters

### PR DESCRIPTION
**Pull request by alimaredia**
This needs to undergo more testing, just want to be able to get feedback on the work that's been done.

Signed-off-by: Ali Maredia <amaredia@redhat.com>

### CODE REVIEW

**Comment by alimaredia** [comment]
retest this please.

**Comment by alimaredia** [comment]
retest this please.

File: tasks/util/rados.py [hunk 1]
```diff
@@ -24,26 +24,26 @@ def rados(ctx, remote, cmd, wait=True, check_status=False):
     else:
         return proc
 
-def create_ec_pool(remote, name, profile_name, pgnum, profile={}):
+def create_ec_pool(remote, name, profile_name, pgnum, profile={}, cluster_name="ceph"):
```
Comment (alimaredia) [inline tasks/util/rados.py:27]: @jdurgin could you look over these changes I made to these pool creation functions so I don't ruin their calls in all the other test suites.

File: tasks/radosgw_admin.py [hunk 2]
```diff
@@ -63,33 +63,33 @@ def remove_newlines(string):
...
-    if multi_region_run:
-        client = rgw_utils.get_master_client(ctx, clients)
+    log.debug('multi_region_run is: %r', multi_region_run)
+    log.debug('master_client is: %r', client)
 
     # once the client is chosen, pull the host name and  assigned port out of
     # the role_endpoints that were assigned by the rgw task
     (remote_host, remote_port) = ctx.rgw.role_endpoints[client]
 
+    realm = None
```
Comment (cbodley) [inline tasks/radosgw_admin.py]: unneeded?

**Review by alimaredia — COMMENTED** [review]
@cbodley here are the runs on teuthology showing that everything doesn't explode:

http://pulpito.ceph.com/amaredia-2016-12-12_20:27:02-rgw:singleton-wip-cbodley-testing---basic-vps/

http://pulpito.ceph.com/amaredia-2016-12-09_23:42:16-rgw:verify-wip-cbodley-testing---basic-vps/

http://pulpito.ceph.com/amaredia-2016-12-09_17:00:56-rgw:multifs-wip-cbodley-testing---basic-vps/

**Comment by liewegas** [comment]
ceph-qa-suite is now merged into ceph.git.  Please resubmit this PR against ceph.git.

**Pull request closed without merging**